cves/CVE-2015-8217.yml

```diff
@@ -167,10 +180,12 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    These developers were listed under "Found By" section of the fix commit.
+    Mateusz "j00ru" Jurczyk and Gynvael Coldwind
```

Comment:
Same as my comment on the other file 
https://j00ru.vexillium.org/2014/01/ffmpeg-and-the-tale-of-a-thousand-fixes/